icons/shields-dot-io.svg

```diff
@@ -0,0 +1 @@
126.0067zm4.2834-1.0666c.4446-.1135.7712-.4309.926-.8997.0522-.1576.0597-.2178.0608-.4838.0014-.3448-.029-.4936-.1553-.7611-.202-.4278-.664-.7017-1.1836-.7017-.6179 0-1.1063.3887-1.2725 1.0127-.0518.1946-.0513.6266.001.8439.1498.622.6527 1.0185 1.2972 1.0227.1057.0007.2525-.0142.3263-.033zm-2.195-1.406v-1.3953h-.3826V13.4603h.3825zm-15.7989.5162l.347-.0304c.0131.2548.1834.5024.4343.5731.2959.1087.6502.097.9174-.0798.1809-.137.2259-.446.0393-.5978-.189-.1604-.4472-.1825-.6767-.2496-.2844-.0686-.5962-.1376-.8024-.3616-.219-.2534-.1975-.6657.0354-.9034.2477-.2564.6338-.3095.9734-.276.2986.0297.614.163.7588.4412.0517.1102.133.272.0797.3772-.1078.021-.2197.018-.3296.0298-.0093-.253-.2096-.4944-.4688-.514-.2309-.0384-.488-.0305-.6907.0974-.1697.12-.2245.4082-.0463.5455.1555.1226.3595.1455.544.1993.3228.076.68.1357.922.3833.2427.2655.2297.7088-.0107.9721-.244.2816-.6438.3608-.9998.3293-.3137-.0178-.6456-.1243-.8394-.3864-.1198-.1564-.1841-.3526-.1871-.549m2.676.893v-2.7802h.3415v.9976c.1688-.209.4479-.3063.7112-.2714.2397.0187.4892.1664.5478.4129.0611.2394.0337.4879.04.733v.9084h-.3414c-.001-.446.0022-.8922-.0017-1.3387-.0031-.163-.076-.3441-.2427-.3984-.2655-.0973-.6162.0477-.6808.3366-.0506.2141-.0262.4352-.032.6531v.7468h-.3415m2.1635-2.3874v-.3925h.3414v.3925zm0 2.3876v-2.014h.3414v2.014zm2.2416-.6486l.3527.0436c-.0852.4204-.5277.7-.9444.6537-.3926-.008-.7804-.2825-.8569-.6794-.1023-.4014-.049-.8842.255-1.1875.374-.3641 1.0558-.3178 1.3636.1082.1668.2246.2034.518.1916.7902h-1.502c-.0173.3506.2751.7282.6488.6867.2283-.0197.4385-.19.491-.4155m-1.1208-.5518h1.1247c.0156-.3367-.3175-.6581-.6577-.572-.264.0518-.4749.3002-.4668.572m1.8967 1.2004v-2.78h.3415v2.78zm2.1787 0v-.2541c-.1714.2964-.5797.3722-.878.2365-.2885-.1238-.4819-.413-.5273-.719-.0615-.335-.015-.7.1686-.9912.2261-.3283.7244-.442 1.0573-.2113.0512.0277.1247.1388.157.1235-.0007-.3215 0-.643-.0003-.9646h.3394v2.7801h-.3167m-1.079-1.005c-.0091.2725.0773.59.3386.7223.2427.1242.5683-.0057.666-.259.109-.2422.0906-.5192.0472-.7746-.0516-.2366-.2484-.4617-.5047-.4618-.231-.0079-.441.1655-.4975.3861-.0397.1248-.0478.2568-.0495.387m1.7986.4043l.3375-.0531c.0153.2051.1837.3937.3931.4084.1697.0257.3654.0146.5001-.1033.098-.0909.1335-.2722.0193-.3653-.1102-.0878-.2567-.0986-.3862-.1423-.2181-.0588-.4507-.1029-.6364-.239-.222-.1883-.2267-.568-.012-.7638.1914-.183.4756-.2166.7283-.1935.2045.0213.4296.0909.5504.269.0563.0868.0859.188.1024.2894l-.3338.0455c-.0147-.1904-.2006-.3352-.3871-.327-.1513-.01-.328.0014-.438.1191-.0757.0879-.0658.2392.0382.3011.121.0857.2754.092.4113.1424.2129.0621.4426.103.624.2385.2046.1791.2097.5151.048.7241-.1575.217-.4365.2934-.6929.2971-.178-.0008-.3622-.026-.5218-.1084-.194-.1103-.3137-.3222-.3446-.5393m6.7365.3107c-.2522-.0395-.525-.2568-.6543-.521-.1443-.2949-.1345-.9031.0195-1.207.1701-.3356.5577-.5235.969-.4696.3576.0468.6359.32.7424.7289.044.1686.0435.561-.0007.7488-.0452.1915-.155.3924-.2676.4897-.127.1096-.3307.2087-.4576.2226-.2166.0238-.2438.0244-.3507.0076z"/></svg>
```

Comment:
Still seeing some differences, even with a linter of 4... 🤔

![image](https://user-images.githubusercontent.com/1830867/103235000-e857da80-4938-11eb-8c1e-2201fecd0046.png)

I've been able to pass the linter (and match source) with this path - any chance you could try that?

```svg
<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Shields.io icon</title><path d="M3.084 13.11a.93.93 0 01-.187-.548l.347-.03c.013.254.183.5.434.572.295.108.65.096.916-.08.18-.137.226-.446.04-.597-.19-.16-.447-.182-.676-.25-.284-.068-.596-.137-.802-.36a.686.686 0 01.036-.903c.247-.256.633-.309.972-.275.298.03.613.162.758.44.051.11.132.272.08.377-.109.02-.22.018-.33.03-.01-.253-.21-.494-.468-.514-.23-.038-.488-.03-.69.098-.17.12-.224.407-.046.544.155.123.359.146.543.2.322.075.679.135.92.382.243.265.23.708-.01.971-.244.281-.643.36-.998.329-.314-.018-.645-.124-.839-.386zm2.827.343v-.746c.006-.217-.018-.438.032-.652.065-.288.415-.433.68-.336.167.054.24.235.243.398.004.446 0 .891.001 1.337h.341v-.907c-.006-.245.021-.493-.04-.732-.058-.247-.307-.394-.547-.413a.787.787 0 00-.71.271v-.996h-.34v2.777h.34zm2.161-2.776h-.341v.392h.341v-.392zm0 .765h-.341v2.012h.341v-2.012zm.451 1.382c-.102-.401-.049-.883.255-1.186.373-.364 1.054-.318 1.362.108.166.224.203.517.19.789h-1.5c-.016.35.276.727.649.686a.56.56 0 00.49-.415l.353.043c-.085.42-.527.7-.943.653-.392-.008-.78-.282-.856-.678zm.327-.57h1.123c.016-.336-.317-.657-.657-.57a.584.584 0 00-.466.57zm2.235-1.577h-.34v2.777h.34v-2.777zm1.835 2.777zm-1.403-.736c-.062-.335-.015-.7.168-.99.226-.328.723-.442 1.056-.211.051.028.124.139.157.123v-.963h.338v2.776h-.316V13.2c-.171.296-.579.371-.877.236-.288-.124-.481-.413-.526-.718zm.325-.268c-.009.272.078.589.339.721a.496.496 0 00.665-.259c.109-.242.09-.518.047-.773-.052-.237-.248-.461-.504-.461a.513.513 0 00-.497.385c-.04.125-.048.257-.05.387zm2.14.942c.16.082.344.107.522.108.256-.004.535-.08.692-.297.161-.209.156-.544-.048-.723-.181-.135-.41-.176-.623-.238-.136-.05-.29-.057-.411-.142-.104-.062-.114-.213-.038-.301.11-.118.286-.129.437-.12.187-.007.372.137.387.328l.333-.046a.748.748 0 00-.102-.289c-.12-.178-.346-.247-.55-.269-.252-.023-.536.011-.727.194a.526.526 0 00.012.762c.185.136.418.18.635.24.13.043.276.054.386.141.114.093.079.274-.02.365-.134.118-.33.129-.499.103-.209-.014-.377-.203-.392-.408l-.337.054v-.001c.03.217.15.429.344.539zm7.102-2.284c-.46-.341-1.219-.134-1.405.417a1.861 1.861 0 00-.082.582c-.046.518.35 1.072.894 1.085.46.027.916-.322.97-.788.095-.452.016-1.005-.377-1.296zM24 9.938v4.122c0 .575-.337 1.072-.823 1.308a.963.963 0 01-.51.146h-6.138l-.041.002c-.02 0-.041 0-.062-.002H1.455A1.456 1.456 0 010 14.06V9.938c0-.802.652-1.454 1.455-1.454H22.667c.187 0 .362.054.51.146.486.236.823.733.823 1.308zm-7.03-.484H1.455a.485.485 0 00-.485.484v4.122c0 .267.217.484.485.484H16.48a.485.485 0 00.489-.445V9.454zm1.714 1.223h-.368v2.777h.368v-2.777zm3.022.642c-.273-.594-1.04-.83-1.633-.618-.572.19-.888.823-.855 1.4 0 .272.07.543.204.78.396.718 1.518.836 2.07.236.443-.474.489-1.227.214-1.798z"/></svg>
```